datasets/glossis/glossis.py

```diff
@@ -117,3 +128,21 @@
 
     # Wait for all the tasks to finish
     wait_gee_tasks(taskids)
+
+
+    # This should result in flowmap tiff files
+    flowmap_task_ids = []
+    flowmap_tiffs = []
+    for current_asset in current_assets:
+        flowmap_tiff = pathlib.Path(current_asset).with_suffix('.tif').name
+        flowmap_tiffs.append(flowmap_tiff)
+        task_id = export_flowmap(current_asset, bucket)
+        flowamp_task_ids.append(task_id)
+    wait_gee_tasks(flowmap_task_ids)
+
+    # This should result in flowmap tiles in a bucket
+    flowmap_tiles_task_ids = []
+    for flowmap_tiff in flowmap_tiffs:
+        tile_dir = generate_wgs84_tiles(flowmap_tiff)
+        upload_dir_to_bucket(bucket_name, source_dir_name=tile_dir, destination_dir_name='flowmaps')
+        # TODO: how do we know which tiles are availabel in backend
```

Comment:
Maybe move the TODO (small typo in there) to an issue and remove here?

Reply:
This is a  separate issue  GDV-324